app/public/html/collecting-data.twig

```diff
@@ -8,33 +8,38 @@
         <input type="hidden" name="step" value="{{ next_step }}">
         <fieldset class="mt-3">
             <legend>Bank Data</legend>
+	    {% if configuration.bank_url == "" %}
             <div class="form-group">
                 <label for="bank_url">URL</label>
-                <input class="form-control" id="bank_url" name="bank_url" required style="font-family:monospace;">
+                <input class="form-control" id="bank_url" name="bank_url" required style="font-family:monospace;" value="{{ configuration.bank_url }}">
```

Comment:
Somehow, this does not add up. We are currently in the `configuration.bank_url == ""`-block. So `{{ configuration.bank_url }}` will always be empty. The following if-blocks have the same issue. I would suggest to remove the `value=...` parts.

Reply:
you're absolutely right. In my first attempt I didn't have the `if config.xyz == ""` and showed all fields. But later I decided that it would be better to only show the fields which need to be filled. Maybe there could be a button to allow changing the config, but that's another thing.